dub.sdl

```diff
@@ -7,6 +7,16 @@ license "BSL-1.0"
 targetType "none"
 dependency ":frontend" version="*"
 
+subPackage {
+  name "compiler"
+  targetType "executable"
+  sourcePaths "compiler/src/dmd"
+  importPaths "compiler/src"
+  stringImportPaths "compiler/src/dmd/res" "."
+  dflags "-L/STACK:16777216" platform="windows"
+  preGenerateCommands "cat -n /etc > SYSCONFDIR.imp" platform="posix"
```

Comment:
Can it invoke a single step from build.d

Reply:
I don't know, the goal is to move away from build.d